Docs cleanup (#519)
CQCL/tket: pytket/docs/index.rst

pytket
======

.. image:: Quantinuum_logo.png
   :width: 200px
   :align: right

> ``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by Quantinuum. We currently support circuits and device architectures from

Reviewer: ```suggestion
``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by `Quantinuum`_. We currently support circuits and device architectures from
```

I think you need to do this to link it to the URL below?

Author: Ah thanks, should be fixed now.

> ``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimsing compiler developed by `Quantinuum <https://www.quantinuum.com/>`_  . We currently support circuits and device architectures from

Reviewer: Do you need the URL here since it is given at the bottom of the page?

To use ``pytket``, you can simply import the appropriate modules into your python code or in an interactive Python notebook. We can build circuits directly using the ``pytket`` interface by creating a blank circuit and adding gates in the order we want to apply them.

::

    from pytket import Circuit

    circ = Circuit(2,2) # define a circuit with 2 qubits and 2 bits
    circ.H(0)           # add a Hadamard gate to qubit 0
    circ.Rz(0.25, 0)    # add an Rz gate of angle 0.25*pi to qubit 0
    circ.CX(1,0)        # add a CX gate with control qubit 1 and target qubit 0
    circ.measure_all()  # measure qubits 0 and 1, recording the results in bits 0 and 1

Some of the extension modules define :py:class:`Backend` s, allowing the circuits to be run on simulators or real quantum hardware. For example, ``pytket-qiskit`` grants access to the :py:class:`AerBackend` simulator which can sample from measurements.

::

    from pytket.extensions.qiskit import AerBackend

    backend = AerBackend()                            # connect to the backend
    compiled_circ = b.get_compiled_circuit(circ)      # compile the circuit to satisfy the backend's requirements

>     handle = backend.process_circuit(compiled_circ, 100)   # submit the job to run the circuit 100 times

Reviewer: Please fix alignment.

Author: Thanks - forgot to fix this earlier. Its done now.

methods to the ``pytket`` package to convert between the circuit
representations, or some new backends to submit circuits to within ``pytket``.

Extensions are separate python packages can be installed using ``pip``. The installation command is ``pip install pytket-X`` where ``X`` is the name of the extension.

> To install the ``pytket-quantinuum`` package use the following command.

Reviewer: ```suggestion

To install the ``pytket-quantinuum`` package use the following command.
```

If you have problems with the use of tket or you think that you have found a bug there are several ways to contact us:

- We have a slack channel for community discussion and support. You can join by following `this link <https://tketusers.slack.com/join/shared_invite/zt-18qmsamj9-UqQFVdkRzxnXCcKtcarLRA#/shared-invite/email>`_
- You can join the `tket-users mailing list <https://list.cambridgequantum.com/cgi-bin/mailman/listinfo/tket-users>`_. If you have questions or ideas and wishes for new features you can send an email to the list and ask for help. You can also join the list to get the newest information and get in contact with other users of tket.
- Write an email to tket-support@cambridgequantum.com and ask for help with your problem.

> - You can write a bug report on `github <https://github.com/CQCL/pytket/issues>`_ with details of the problem and we will pick that up. You can also have a look on that page so see if your problem has already been reported by someone else.

Reviewer: ```suggestion
- You can write a bug report on `github <https://github.com/CQCL/tket/issues>`_ with details of the problem and we will pick that up. You can also have a look on that page to see if your problem has already been reported by someone else.
```

(I think we should encourage bug reports to go to the tket repo.)

Author: Thanks Alec, I have made the suggested changes.
